symphony/bdk/ext/group.py

```diff
@@ -0,0 +1,149 @@
+            the first page
+        :param limit: maximum number of items to return
+        :param sort_order: sorting direction of items (ordered by creation date)
+        :return: the list of matching groups
+        """
+        kwargs = dict(x_symphony_host="", type_id="SDL")
+        if status is not None:
+            kwargs["status"] = status
+        if before is not None:
+            kwargs["before"] = before
+        if after is not None:
+            kwargs["after"] = after
+        if limit is not None:
+            kwargs["limit"] = limit
+        if sort_order is not None:
+            kwargs["sort_order"] = sort_order
+        return await self._group_api.list_groups(**kwargs)
+
+    async def update_group(self, if_match: str, group_id: str, update_group: UpdateGroup) -> ReadGroup:
+        """Update an existing group
+        See: `Update a group <https://developers.symphony.com/restapi/reference/updategroup>`_
+
+        :param if_match: eTag of the group to be updated
+        :param group_id: the ID of the group
+        :param update_group: the group fields to be updated
+        :return: the updated group
+        """
+        return await self._group_api.update_group(x_symphony_host="", if_match=if_match, group_id=group_id,
+                                                  update_group=update_group)
+
+    async def update_avatar(self, group_id: str, image: str) -> ReadGroup:
+        """Update the group avatar
+        See: `Update the group avatar <https://developers.symphony.com/restapi/reference/updateavatar>`_
+
+        :param group_id: the ID of the group
+        :param image: The avatar image for the user profile picture.
+            The image must be a base64-encoded .jpg, .png, or .gif. Image size limit: 2 MB
+        :return: the updated group
+        """
+        upload_avatar = UploadAvatar(image=image)
+        return await self._group_api.update_avatar(x_symphony_host="", group_id=group_id, upload_avatar=upload_avatar)
+
+    async def get_group(self, group_id: str) -> ReadGroup:
+        """Retrieve a specific group
+        See: `Retrieve a group <https://developers.symphony.com/restapi/reference/getgroup>`_
+
+        :param group_id: the ID of the group to retrieve
+        :return: the group details
+        """
+        return await self._group_api.get_group(x_symphony_host="", group_id=group_id)
+
+    async def add_member_to_group(self, group_id: str, user_id: int) -> ReadGroup:
+        """Add a new user to an existing group.
+        See: `Add a new user to an existing group <https://developers.symphony.com/restapi/reference/addmembertogroup>`_
+
+        :param group_id: the ID of the group in which to add the user
+        :param user_id: The ID of the user to be added into the group
+        :return: the updated group
+        """
+        member = Member(member_id=user_id, member_tenant=extract_tenant_id(user_id))
```

Comment:
Mmm I missed that in the Java BDK we were doing that (guessing the tenant id from the user id) (which the profile manager API should do for us already...)
So I should probably change the WDK implem too (although we only do it partially for this method and not for the creation)